test/unit/processing/statement/get-statement.test.js

```diff
@@ -72,8 +82,19 @@ describe('get various components and transform to statement object', () => {
 
   test('should call getCalculation when a paymentRequestId is given', async () => {
     const paymentRequestId = 1
-    const statement = await getStatement(paymentRequestId)
-    console.log(statement)
+    await getStatement(paymentRequestId)
     expect(getCalculation).toHaveBeenCalled()
   })
+
+  test('should call getLastSettlement when a paymentRequestId is given', async () => {

```

Comment:
Can you add tests for:

- called who many times
- called with whatever parameters

Reply:
Yes, think we need to think about adding a bit more for all the other method calls too.  There was a lot of setup in this to not really verify a lot.